src/core/mac/mac_frame.cpp

```diff
@@ -363,6 +471,9 @@ bool Frame::IsDstPanIdPresent(uint16_t aFcf)
         present = IsDstAddrPresent(aFcf);
     }
 
```

Comment:
Can cosnider adding `ExitNow()` at the end of method. 
This way we don't need to add `#if OPENTHREAD_CONFIG_WAKE_UP_COORDINATOR_ENABLE || OPENTHREAD_CONFIG_WAKE_UP_END_DEVICE_ENABLE` checks around the `exit` label?